test/common/user_rhmi_developer_permissions.go

```diff
@@ -207,3 +208,29 @@ func verifyRHMIDeveloperRHMIConfigPermissions(t TestingTB, openshiftClient *reso
 
 	verifyCRUDLPermissions(t, openshiftClient, expectedPermission)
 }
+
+func verifyRHMIDeveloperRHMIPermissions(t TestingTB, openshiftClient *resources.OpenshiftClient) {
+	t.Log("Verifying RHMI Developer permissions for RHMI Resource")
+
+	expectedPermission := ExpectedPermissions{
+		ExpectedCreateStatusCode: 403,
+		ExpectedReadStatusCode:   403,
+		ExpectedUpdateStatusCode: 403,
+		ExpectedDeleteStatusCode: 403,
+		ExpectedListStatusCode:   403,
+		ListPath:                 fmt.Sprintf(resources.PathListRHMI, RHMIOperatorNamespace),
+		GetPath:                  fmt.Sprintf(resources.PathGetRHMI, RHMIOperatorNamespace, "rhoam"),
+		ObjectToCreate: &integreatlyv1alpha1.RHMI{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "rhoam",
```

Comment:
In "object create test" we need to have a different CR name than the one that already exists on a cluster after installation of RHOAM (`rhoam`)
```suggestion
				Name:      "test-rhoam",
```